features.json

```diff
@@ -5,6 +5,7 @@
     "ember-libraries-isregistered": null,
     "ember-application-engines": true,
     "ember-glimmer": null,
+    "ember-glimmer-allow-two-way-reflush": true,
```

Comment:
This is the thing I wanted to ask you 😄

1. The detailed reporting should be disabled on prod no matter what (`lastRenderedFrom` and friends – what @chadhietala wrapped in `runInDebug` currently)
2. By default, what is currently a deprecation in the code "You modified ${parts.join('.')} twice in a single render..." should be *an error* in development mode, with a more detailed message: it should describe the situation more clearly, that it is previously unreliable and very slow, and probably link to somewhere that tells you more including how to fix it. It should also mention that you can enable the feature with an addon, see (4) and (5).
3. By default, the `lastRendered` tracking should be off (what is currently in the "off" branches of "ember-glimmer-allow-two-way-reflush"
4. If the compatibility mode is on, in development mode, it should still emit the message as a deprecation.
5. If the compatibility mode is on, in production mode, it should work without a deprecation (which is the natural consequence of a deprecation I believe).